### PR TITLE
MODNOTIFY-55: mod-notify is not passing the X-Okapi-Request-Id header to its API calls

### PR DESCRIPTION
**Pull request by yusuf-murodov-epam**
## Purpose
[MODNOTIFY-55](https://issues.folio.org/browse/MODNOTIFY-55)
_mod-notify is not passing the X-Okapi-Request-Id header to its API calls_

### CODE REVIEW

File: src/main/java/org/folio/client/NoticesClient.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
+package org.folio.client;
+
+import static javax.ws.rs.core.HttpHeaders.*;
```
Comment (sherzodn) [inline src/main/java/org/folio/client/NoticesClient.java:3]: try to import only used classes

File: src/main/java/org/folio/client/NoticesClient.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  }
+
+  public Future<EventEntity> getEventConfig(String name) {
+    Promise<HttpResponse<Buffer>> promise = Promise.promise();
+    getAbs("/eventConfig")
+      .addQueryParam("query", "name==" + name)
+      .putHeader(ACCEPT, APPLICATION_JSON)
+      .send(promise);
+
+
```
Comment (sherzodn) [inline src/main/java/org/folio/client/NoticesClient.java:35]: remove the excess line

File: src/main/java/org/folio/client/OkapiClient.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+  protected static <T> Function<HttpResponse<Buffer>, T> responseMapper(Class<T> type) {
+    return resp -> {
+      if (resp.statusCode() == SC_BAD_REQUEST) {
+        throw new BadRequestException(resp.bodyAsString());
+      }
+      if (resp.statusCode() != SC_OK && resp.statusCode() != SC_NO_CONTENT) {
+        throw new InternalServerErrorException();
+      }
+      return type == Void.class ? null : resp.bodyAsJson(type);
```
Comment (sherzodn) [inline src/main/java/org/folio/client/OkapiClient.java:65]: null return is not good practice

File: src/main/java/org/folio/client/NoticesClient.java (hunk 1 above)
Comment (yusuf-murodov-epam) [reply src/main/java/org/folio/client/NoticesClient.java]: fixed

File: src/main/java/org/folio/client/NoticesClient.java (hunk 2 above)
Comment (yusuf-murodov-epam) [reply src/main/java/org/folio/client/NoticesClient.java]: removed

File: NEWS.md [hunk 4]
```diff
@@ -1,3 +1,11 @@
+## 2.10.0 2022-02-22
```
Comment (sherzodn) [inline NEWS.md:1]: this release notes here?

File: pom.xml [hunk 5]
```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>mod-notify</artifactId>
-  <version>2.10.0-SNAPSHOT</version>
+  <version>2.11.0-SNAPSHOT</version>
```
Comment (sherzodn) [inline pom.xml:6]: why did these release notes add as new changes? have you used git rebase master?

**Pull request closed without merging**